### PR TITLE
This is my submission for the openQA FirewallD test.

### PR DESCRIPTION
**Pull request by atanno**
- Related ticket: https://progress.opensuse.org/issues/47159
- Needles: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/xyz
- Verification run: http://10.100.51.204/tests/59#
- Verification run: http://10.100.51.204/tests/60# ( using systemctl  is-active firewalld )

### CODE REVIEW

File: tests/console/firewalld.pm [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+use base "consoletest";
+use testapi;
+use transactional_system 'trup_install';
+
+sub run {
+    select_console("root-console");
+
+    # Check Service State, enable it if necessary
+    record_info 'Check Service State';
+    assert_script_run("if [ `firewall-cmd --state > /dev/null 2>&1 ; echo $?` -eq 252 ]; then systemctl start firewalld; fi");
```
Comment (mimi1vx) [inline tests/console/firewalld.pm]: why not `systemctl is-active firewalld`  ?

File: tests/console/firewalld.pm [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+# without any warranty.
+
+# Summary: Test FirewallD basic usage
+# Maintainer: Alexandre Makoto Tanno <atanno@suse.com>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use transactional_system 'trup_install';
```
Comment (mimi1vx) [inline tests/console/firewalld.pm]: `use transactional_system 'trup_install';` why is this here ?

Comment (atanno) [reply tests/console/firewalld.pm]: Hmmm, this came from another test I used as a template. I will remove this.

File: tests/console/firewalld.pm (hunk 1 above)
Comment (atanno) [reply tests/console/firewalld.pm]: I can change this as well.

Comment (atanno) [reply tests/console/firewalld.pm]: I've modified the test to use `systemctl is-active firewalld`.

File: tests/console/firewalld.pm (hunk 2 above)
Comment (atanno) [reply tests/console/firewalld.pm]: Removed 'use transactional_system 'trup_install'' ' from the test.
